ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/point/DropPinTaskMapFragment.kt

```diff
@@ -44,11 +42,6 @@ class DropPinTaskMapFragment(private val viewModel: DropPinTaskViewModel) :
     savedInstanceState: Bundle?,
   ): View {
     val root = super.onCreateView(inflater, container, savedInstanceState)
```

Comment:
We can cleanup this overridden method as well.

Reply:
Thank you! Cleaned up nicely!